app/src/main/kotlin/no/nav/tiltakspenger/vedtak/clients/meldekort/MeldekortTilBrukerDto.kt

```diff
@@ -0,0 +1,86 @@
+package no.nav.tiltakspenger.vedtak.clients.meldekort
+
+import no.nav.tiltakspenger.meldekort.domene.Meldekort
+import no.nav.tiltakspenger.meldekort.domene.MeldekortStatus
+import no.nav.tiltakspenger.meldekort.domene.Meldekortdag
+import no.nav.tiltakspenger.meldekort.domene.Meldekortdag.IkkeUtfylt
+import no.nav.tiltakspenger.meldekort.domene.Meldekortdag.Utfylt.Deltatt.DeltattMedLønnITiltaket
+import no.nav.tiltakspenger.meldekort.domene.Meldekortdag.Utfylt.Deltatt.DeltattUtenLønnITiltaket
+import no.nav.tiltakspenger.meldekort.domene.Meldekortdag.Utfylt.Fravær.Syk.SykBruker
+import no.nav.tiltakspenger.meldekort.domene.Meldekortdag.Utfylt.Fravær.Syk.SyktBarn
+import no.nav.tiltakspenger.meldekort.domene.Meldekortdag.Utfylt.Fravær.Velferd.VelferdGodkjentAvNav
+import no.nav.tiltakspenger.meldekort.domene.Meldekortdag.Utfylt.Fravær.Velferd.VelferdIkkeGodkjentAvNav
+import no.nav.tiltakspenger.meldekort.domene.Meldekortdag.Utfylt.IkkeDeltatt
+import no.nav.tiltakspenger.meldekort.domene.Meldekortdag.Utfylt.Sperret
+import java.time.LocalDate
+
+// TODO abn: Dette er veldig work in progress
+// Bør inn i felles libs når vi får landet en modell, brukes også i meldekort-api
+
+enum class MeldekortStatusTilBruker {
+    KAN_UTFYLLES,
+    KAN_IKKE_UTFYLLES,
+    GODKJENT,
+}
+
+enum class MeldekortDagStatusTilBruker {
```

Comment:
Vi kan godt ha dette som en første versjon, men jeg tenker vi kan sette oss ned og splitte denne opp i 2 modeller. En for meldeperioder (kun det som trengs for å fylle ut) og en for godkjente meldekort

Reply:
På over ett års sikt kan vi vurdere om vi også ønsker en eventuell tredje modell: veileder-utfylte rapporteringer som er til godkjenning